lib/conman.rb

```diff
@@ -43,6 +48,20 @@ def total_contacts
     contact_list.size
   end
 
+  def search_contacts
+    search_again = true
+
+    while search_again == true
+      ui.display_all(search_contact)
+      search_again = ui.ask_search_again
+    end
+  end
+
+  def search_contact
```

Comment:
Can you identify all responsibilities of `search_contact`? Looking at the implementation it does more than one thing.